src/components/App.jsx

```diff
@@ -146,8 +146,8 @@ export default function App() {
             <rb.Card.Body>
               <rb.Card.Title className="text-center mb-3">Warning</rb.Card.Title>
               <p className="text-secondary">
-                While JoinMarket is tried and tested, this user interface is not. It is in a pre-alpha stage and
-                currently does not offer the same functionality and privacy guarantees as existing JoinMarket tools.
+                While JoinMarket is tried and tested, Jam is not. It is in an alpha stage and currently does not offer
+                the same functionality and privacy guarantees as existing JoinMarket tools.
```

Comment:
```suggestion
                While JoinMarket is tried and tested, Jam is not. It is in an alpha stage, so use with caution.```

we do have [almost] feature parity, and not sure why there would be difference in privacy guarentees.

Reply:
You're right! This is just some ad-hoc text we threw in before the first release since we haven't really tested it on mainnet and don't support the tumbler yet (that's what the "privacy guarantees" hints at IIRC since you can only do single CoinJoins at the moment). We can definitely update/change it, though.

Do you have something in mind that would fit better?

Reply:
Sorry somehow missed your inline suggestion. Not sure how. 😅 Applied just now. ✅